src/build.d

```diff
@@ -84,32 +93,25 @@ Command-line parameters
         return;
     }
 
-    if (res.helpWanted)
```

Comment:
Moved lower, because now printing help depends on targets being defined, which depend on the environment being setup.